qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -32,74 +32,76 @@
 class FineAmplitude(BaseExperiment):

```

Comment:
Can you add a reference to this class in `FineXAmplitude` and `FineSXAmplitude` overview?

Reply:
See here: 2f0daf1